tests/template_test.go

```diff
@@ -13,37 +13,42 @@ import (
 )
 
 func TestTemplates(t *testing.T) {
+	blackListedTests := os.Getenv("BLACK_LISTED_TESTS")
+	if blackListedTests == "" {
+		t.Skip("Skipping tests due to missing `BLACK_LISTED_TESTS` env var")
```

Comment:
Alternatively, we could assume it to be empty

Reply:
yeah i am fixing that right now :)